bundles/nl.asml.matala.product/src/nl/asml/matala/product/Product.xtext

```diff
@@ -64,6 +59,7 @@ Block:
 			('outputs' outvars+=Variable* )? &
 			('local' localvars+=Variable* )? &
 			('init' initActions+=(AssignmentAction | RecordFieldAssignmentAction)*)? &
+			('sut-var' '{' sutVar += [expr::Variable|ID]* '}')? &
```

Comment:
@dbera This is different than what the bpmn4s2pspec transformation produces, that requires a double colon and omits the curly brackets. What do we want to use here? I propose to omit both the double colon and the curly brackets and align more with the syntax of defining `produces-outputs`.